MobiFlight/Config/Config.cs

```diff
@@ -155,6 +155,8 @@ public Config FromInternal(String value, bool throwException = false)
                                 // (in this case InputMultiplexer) will provide them
                                 // Treat the MultiplexerDriver as a regular device (add it to the items list), except it won't be shown in the GUI tree.
                                 Items.Add(multiplexerDriver);
+                            } else {
```

Comment:
I would like to keep the symmetry here with calling .FromInternal() for the MultiplexerDriver.
What stands out to me is that we interact with the `registerClient` method from the outside (here), and the inside (inside FromInternal)

``` C#
// Build multiplexerDriver if none found yet 
if (multiplexerDriver == null) {
    multiplexerDriver = new MobiFlight.Config.MultiplexerDriver();
    Items.Add(multiplexerDriver);
}
// then we can always call 
multiplexerDriver.FromInternal(item + BaseDevice.End);
```

Inside from internal the "registerClient" is called, and the pins can be set, if required (we can do it once, or we can do it every time, it doesn't matter if the config was created through mobiflight the base pins will be the same)

Reply:
This would definitely work; calling `MultiplexerDriver.FromInternal()` from here (rather than from `WhateverClient.FromInternal()`) would apparently simplify the code.

The only drawback is that the client should have a method to supply the part of its config string to be passed to `multiplexerDriver.FromInternal()` (in the correct format); therefore I had to add a `GetMultiplexerDriverConfig()` method, any other suggestion welcome.

Unfortunately, this now breaks the symmetry with the creation of the corresponding objects in `MobiFlightPanel`, where the call to `registerClient()` is explicit (there are no methods to hide it in). 

On a related note, I still stand by my opinion that having the Multiplexer driver defined in its own specific configuration line only would be a better choice.
The current situation, besides being dangerous because of the redundant definition of the same object in several places (which is required and assumed - without any enforcement or check in place - to be consistent), also brings to additional needless clutter in the configuration file, which will only get worse if further mux client devices are ever implemented. As for the advantages, I fail to see any, but I'd be glad to see it proven otherwise.
(BTW, the above mentioned `GetMultiplexerDriverConfig()` method would also become unnecessary - another simplification.)